src/greentest/test__greenlet.py

```diff
@@ -699,6 +699,39 @@ def func():
         while not raw.dead:
             gevent.sleep(0.01)
 
+    def test_add_spawn_callback(self):
+        called = 0
+
+        def cb(gr):
+            nonlocal called
```

Comment:
We support Python 2 still, where [nonlocal is a syntax error](https://travis-ci.org/gevent/gevent/jobs/441913993#L678).

Reply:
Right. I've pushed a fix.